specs/commandline/sign.md

```diff
@@ -42,6 +42,8 @@ Flags:
        --plugin string              signing plugin name. This is mutually exclusive with the --key flag
        --plugin-config stringArray  {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values.
        --signature-format string    signature envelope format, options: "jws", "cose" (default "jws")
+       --tsa-root-cert string       filepath of timestamp authority root certificate
```

Comment:
How about `--tsa-cert-chain` filepath to the certificate chain file for the RFC3161 Timestamping Authority (TSA). Must contain the root CA certificate. Optionally it may contain intermediate CA certificates.

Reply:
@yizha1 Please see my research on TSA servers below: https://github.com/notaryproject/notation/pull/978#issuecomment-2229929172

The TSAs are already including their intermediate certs in the response, I'm not seeing any extra benefit of user providing them again. Therefore, in this iteration, I would propose only require user to provide the TSA root cert.